cmd/riscli/dump_loc_rib.go

```diff
@@ -52,7 +73,11 @@ func dumpRIB(c pb.RoutingInformationServiceClient, routerName string, vrfID uint
 	for {
 		r, err := client.Recv()
 		if err != nil {
-			return errors.Wrap(err, "Received failed")
+			if err == io.EOF {
+				return nil
+			} else {
```

Comment:
This means we're returning an error when err != io.EOF.
Make the else branch a separate if.